packages/contentstack-import/src/lib/import/entries.js

```diff
@@ -42,8 +42,10 @@ let masterLanguage;
 
 let skipFiles = ['__master.json', '__priority.json', 'schema.json'];
 let entryBatchLimit = config.rateLimit || 10;
+const importConcurrency = eConfig.importConcurrency || config.importConcurrency;
+const writeConcurrency = eConfig.writeConcurrency || config.writeConcurrency;
 
-function importEntries() {
+function EntriesImport() {
```

Comment:
Are we not changing entries module from functional programming to OOPs.?

Reply:
No really required for now